Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -1882,6 +1883,9 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
     void handleControlledElementIDResponse(const String&) const;
     bool isPlayingVideoInEnhancedFullscreen() const;
 
+    RetainPtr<STWebpageController> screenTimeWebpageController() const { return m_screenTimeWebpageController; };
+    void setScreenTimeWebpageController(RetainPtr<STWebpageController> st) { m_screenTimeWebpageController = st; }
```

Comment:
Minor nit - we generally avoid abbreviations like `st`, even in cases like this.
```suggestion
    void setScreenTimeWebpageController(RetainPtr<STWebpageController>&& controller) { m_screenTimeWebpageController = WTFMove(controller); }
```